src/proposals/20240417/SparkEthereum_20240417.sol

```diff
@@ -26,6 +27,12 @@ contract SparkEthereum_20240417 is SparkPayloadEthereum {
     address public constant POOL_IMPLEMENTATION_NEW = 0x5aE329203E00f76891094DcfedD5Aca082a50e1b;
     address public constant FREEZER_MOM_NEW         = 0x237e3985dD7E373F2ec878EC1Ac48A228Cf2e7a3;
     address public constant FREEZER_MULTISIG        = 0x44efFc473e81632B12486866AA1678edbb7BEeC3;
+
+    // Formula for 10% target DSR APY (0.095310179948351216930400000)
+    // bc -l <<< 'scale=27; (1.000000003022265980097387650 - 1) * 60 * 60 * 24 * 365'
```

Comment:
For future reference this should match formula below using `l(1.10)`

Reply:
But we are targetting 11% APY, not 10%, so it should be correct. `bc -l <<< 'scale=27; (e( l(1.11)/(60 * 60 * 24 * 365) ) - 1) * 60 * 60 * 24 * 365 - 0.095310179948351216930400000'` == correct APR spread for conversion of 10% DSR to 11% borrow.